prime-gaming.js

```diff
@@ -346,49 +347,65 @@ try {
-          unlinked_store = 'epic-games';
-        }
-        if (unlinked_store) {
-          console.error('  Missing account linking:', unlinked_store, url);
-          dlc_unlinked[unlinked_store] ??= [];
-          dlc_unlinked[unlinked_store].push(title);
+          unlinked_store = await linkAccountButton.getAttribute("aria-label");
+          console.debug(`  Not able to claim. ${unlinked_store}`);
+        // console.debug('  LinkAccountButton label:', unlinked_store);
+        // const match = unlinked_store.match(/Link (.*) account/);
+        // if (match && match.length == 2) unlinked_store = match[1];
+      // } else if(await page.locator('text=Link game account').count()) { // epic-games only?
+      //   console.error('  Missing account linking (epic-games specific button?):', await page.locator('button[data-a-target="gms-cta"]').innerText()); // TODO needed?
+      //   unlinked_store = 'epic-games';
+      // }
+      // if (unlinked_store) {
+      //   console.error('  Missing account linking:', unlinked_store, url);
+      //   dlc_unlinked[unlinked_store] ??= [];
+      //   dlc_unlinked[unlinked_store].push(title);
+      // } else {
+      //   const code = await page.inputValue('input[type="text"]');
+      //   console.log('  Code to redeem game:', code);
+      //   db.data[user][title].code = code;
+      //   db.data[user][title].status = 'claimed';
+      //   // notify_game.status = `<a href="${redeem[store]}">${redeem_action}</a> ${code} on ${store}`;
+      // }
+      // await page.pause();
         } else {
-          const code = await page.inputValue('input[type="text"]');
-          console.log('  Code to redeem game:', code);
-          db.data[user][title].code = code;
-          db.data[user][title].status = 'claimed';
-          // notify_game.status = `<a href="${redeem[store]}">${redeem_action}</a> ${code} on ${store}`;
+          //TODO this needs to be tested
+          //ran out of possible prime offers
+          // const code = await page.inputValue('input[type="text"]', {
+          //   timeout: 1000,
+          // });
+
+          // if (code) {
+          //   console.log("  Code to redeem game:", code);
+          // } else {
+          console.log("  Claimed!");
+          // }
         }
-        // await page.pause();
       } catch (error) {
         console.error(error);
       } finally {
-        await page.goto(URL_CLAIM, { waitUntil: 'domcontentloaded' });
+        await page.goto(URL_CLAIM, { waitUntil: "domcontentloaded" });
         await page.click('button[data-type="InGameLoot"]');
       }
     }
-    console.log('DLC: Unlinked accounts:', dlc_unlinked);
+    // TODO
+    // console.log("DLC: Unlinked accounts:", dlc_unlinked);
   }
 } catch (error) {
   process.exitCode ||= 1;
-  console.error('--- Exception:');
+  console.error("--- Exception:");
```

Comment:
There are still unnecessary changes here

Reply:
should have changed everything. if i still forgot something let me know